.github/workflows/build_package.yml

```diff
@@ -122,23 +122,23 @@ jobs:
             build-family: linux-x86_64
             build-package: main-dist-linux
             experimental: false
-          - runs-on: ah-ubuntu_22_04-c7g_4x-50
+          - runs-on: ubuntu-24.04-arm
```

Comment:
Did you test these runners, or would we wait until tonight's release to see if they work? The installed software may be different, though we do build under Docker: https://github.com/iree-org/iree/blob/f1a20c3156771baf569543f78e52755985ec1270/build_tools/python_deploy/build_linux_packages.sh#L67

(we should actually pin `quay.io/pypa/manylinux_2_28_aarch64` so it doesn't pull the latest `main` 🤔 )